addons/website_sale/static/src/interactions/website_sale_reorder.js

```diff
@@ -0,0 +1,66 @@
+import { _t } from '@web/core/l10n/translation';
+import { rpc } from '@web/core/network/rpc';
+import { registry } from '@web/core/registry';
+import { ConfirmationDialog } from '@web/core/confirmation_dialog/confirmation_dialog';
+import { Interaction } from '@web/public/interaction';
+
+export class SaleOrderPortalReorder extends Interaction {
+    static selector = ".o_portal_sidebar";
+    dynamicContent = {
+        'button.o_wsale_reorder_button': { 't-on-click': async (ev) => await this._onReorder(ev) },
```

Comment:
No need to wrap the method (but I think you need to make it public) :slightly_smiling_face: 
```suggestion
        'button.o_wsale_reorder_button': { 't-on-click': this.onReorder },
```

Reply:
Sure!